pkgs/by-name/ch/chuffed/package.nix

```
@@ -0,0 +1,38 @@
+{
+  stdenv,
+  cmake,
+  fetchFromGitHub,
+  lib,
+}:
+stdenv.mkDerivation {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
with a fixed-point function ofc